src/adapter/src/coord/ddl.rs

```diff
@@ -437,8 +437,10 @@ impl Coordinator {
                     event.object_type.as_title_case(),
                     event.event_type.as_title_case()
                 );
+                let application_name = session.map(|s| s.application_name()).unwrap_or("");
```

Comment:
Can you confirm this, I'm pretty sure that when `session` is `None` that means that the transaction was initiated by the system and not the user. In that case maybe `"environmentd"` or some other string would be more meaningful than an empty string.

Reply:
@RobinClowers or someone from @MaterializeInc/cloud may have insight into this

Reply:
Hmm, I don't know the answer to that. When you say "the system", are we talking about internal environmentd things, or cloud components (either the controllers or the promsql exporter)?

Reply:
Internal envd things. Cloud components still have to login with a username.

Reply:
Sounds good to me! I updated the default name to be `environmentd`